requirements/default.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.15.1
 scipy>=1.0.1
-matplotlib>=2.0.0,!=3.0.0
+matplotlib>=3.3,!=3.0.0
```

Comment:
```suggestion
matplotlib>=3.3
```

Reply:
Would you accept this change and would you like to rebase @stefanv ?

I'm OK to bump for a 6 months old version.